custom/icds_reports/ucr/data_sources/migrations_form.json

```diff
@@ -143,21 +143,25 @@
       },
       {
         "column_id": "migration_status",
-        "datatype": "string",
+        "datatype": "small_integer",
```

Comment:
I believe this would be better to leave like string for the case as handling this migration would be tedious. The value could be just either `migrated` or null. So aggregation just check for value `migrated`.  @calellowitz ?

Reply:
Depending on how many migration forms there are, the ideal might be to make the change then drop everything and reindex from scratch. If there are too many forms we can make a new column that holds this as an integer, and leave the existing column but not use it. Either way, an integer is preferable because it keeps this table consistent with other UCRs

Reply:
My preference would to be then to create a new column. COol. THanks,

Reply:
@ShubhamBansal1997  could you create a new column in the UCR which will have value as Integer and use that further in aggregation instead of this existing one.  you can name it as is_migrated something
I think we can also remove this existing column(please wait, I am confirming something). 

But you can create a new one holding ints and use that in agg.